cmd/snap-confine/snap-confine.c

```diff
@@ -693,6 +693,7 @@ static void enter_non_classic_execution_environment(sc_invocation * inv,
 		if (unshare(CLONE_NEWNS) < 0) {
 			die("cannot unshare the mount namespace");
 		}
+		sc_store_ns_fstab(inv);
```

Comment:
should we do this if we are not in normal_mode? I wonder if this should be instead closer to the point where we make the / tmpfs?

Reply:
Right, I moved it to right where we setup the tmpfs.